docs/index.md

```diff
@@ -42,6 +39,12 @@
 * [Introduction to LESS](./frontend/introduction-to-less.md)
 * [Frontend Troubleshooting](./frontend/frontend-troubleshooting.md)
 
+## Orchestration
```

Comment:
Is this list of all relevant Kubernetes docs? I am missing some e.g. "Deploy your application to Google Cloud on your CI/CD". But maybe they are not so important.

Reply:
You are right, I added CI deploy section into it. It would lead you there by the other articles but iguess its better to have it here too..